woocommerce/woocommerce-ios
# Added line break between "Address 1" and "Address 2" in Order Detail

Closes: #7982 

## Description
I added a line break between address 1 and address 2 in Order Detail. Before they were displayed on the same line, so they appear like a unique address.

## Testing instructions
Fill both "Address 1" and "Address 2" of the order's Shipping details, save, and go back to the order detail screen. You should see Address 1 on line 1, and Address 2 on a new line.


## Screenshots
<img src="https://user-images.githubusercontent.com/495617/223178765-dc5845fd-2345-4de1-9167-d2b6ce07bc2b.png" width=300 />



---
